README.md

```diff
@@ -48,11 +48,17 @@ var childWalletSecret = lib.derivePublic(parentWalletPublicKey, 1, 1)
 * `Buffer toPublic(Buffer privateKey)`
 * `Buffer decodePaperWalletMnemonic(string paperWalletMnemonic)`
 * `Buffer xpubToHdPassphrase(Buffer xpub)`
-* `string packAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer deriveBootstrapAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer deriveBaseAddress(Buffer pubKey, Buffer stakePubKey, int addressTypeId, int protocolMagic, Bool isStakeHash = false)`
+* `Buffer derivePointerAddress(Buffer pubKey, Object pointer, int addressTypeId, int protocolMagic)`
+* `Buffer deriveEnterpriseAddress(Buffer pubKey, int addressTypeId, int protocolMagic)`
+* `Object getAddressInfo(Buffer address)`
 * `string unpackAddress(string address, Buffer hdPassphrase)`
 * `Bool isValidAddress(string address)`
 * `Buffer blake2b(Buffer input, outputLen)`
 * `Buffer cardanoMemoryCombine(Buffer input, String password)`
+* `string bech32_encode(string prefix, Buffer data)`
```

Comment:
I would use camelcase for the sake of consistency and it makes sense to expose the base58 API in the same way for the sake of consistency